rfc5424logging/handler.py

```diff
@@ -230,127 +237,122 @@ def emit(self, record):
-            cleaned_structured_data = []
-            for sd_id, sd_params in list(structured_data.items()):
-                # Clean structured data ID
-                sd_id = self.filter_printusascii(sd_id)
-                sd_id = sd_id.replace('=', '').replace(' ', '').replace(']', '').replace('"', '')
-                if '@' not in sd_id and sd_id not in REGISTERED_SD_IDs and enterprise_id is None:
-                    raise ValueError("Enterprise ID has not been set. Cannot build structured data ID. "
-                                     "Please set a enterprise ID when initializing the logging handler "
-                                     "or include one in the structured data ID.")
-                elif '@' in sd_id:
-                    sd_id, enterprise_id = sd_id.rsplit('@', 1)
-
-                if len(enterprise_id) > 30:
-                    raise ValueError("Enterprise ID is too long. Impossible to build structured data ID.")
-
-                sd_id = sd_id.replace('@', '')
-                if len(sd_id) + len(enterprise_id) > 32:
-                    sd_id = sd_id[:31 - len(enterprise_id)]
-                sd_id = '@'.join((sd_id, enterprise_id))
-                sd_id = sd_id.encode('ascii', 'replace')
-
-                cleaned_sd_params = []
-                # ignore sd params not int key-value format
-                if isinstance(sd_params, dict):
-                    sd_params = sd_params.items()
-                else:
-                    sd_params = []
-
-                # Clean key-value pairs
-                for (param_name, param_value) in sd_params:
-                    param_name = self.filter_printusascii(str(param_name))
-                    param_name = param_name.replace('=', '').replace(' ', '').replace(']', '').replace('"', '')
-                    if param_value is None:
-                        param_value = ''
-
-                    param_value = str(param_value)
-
-                    if PY2:
-                        param_value = unicode(param_value, 'utf-8')  # noqa
-
-                    param_value = param_value.replace('\\', '\\\\').replace('"', '\\"').replace(']', '\\]')
-
-                    param_name = param_name.encode('ascii', 'replace')[:32]
-                    param_value = param_value.encode('utf-8', 'replace')
-
-                    sd_param = b''.join((param_name, b'="', param_value, b'"'))
-                    cleaned_sd_params.append(sd_param)
-
-                cleaned_sd_params = SP.join(cleaned_sd_params)
-
-                # build structured data element
-                spacer = SP if cleaned_sd_params else b''
-                sd_element = b''.join((b'[', sd_id, spacer, cleaned_sd_params, b']'))
-                cleaned_structured_data.append(sd_element)
-
-            if cleaned_structured_data:
-                structured_data = b''.join(cleaned_structured_data)
+        pri = '<%d>' % self.encodePriority(self.facility,
```

Comment:
All of these changes are not really changes at all except I took them out of the try/except they were in, as any exceptions raised will be handled by the exception handler above.